pkg/transform/internal/test/load_test.go

```diff
@@ -0,0 +1,9 @@
+package test
+
+import (
+	"testing"
+)
+
+func TestTest(t *testing.T) {
+	t.Skip()
```

Comment:
What was the reason for adding this test?

Reply:
It shows that we care about test but this one doesn't have much coverage.